lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer.go

```diff
@@ -89,13 +94,77 @@ func (s *subscriberdbServicer) ListSubscribers(ctx context.Context, req *lte_pro
 		subProto.NetworkId = &protos.NetworkID{Id: networkID}
 		subProtos = append(subProtos, subProto)
 	}
+
+	// Short-circuit subscriber data transmission by comparing cloud & gateway digests.
+	// If digest generation fails, the error is swallowed to not affect the main functionality.
+	//
+	// Note: currently the logic only applies to a request for the first page to avoid concurrency issues;
+	// more to be implemented with gateway subscriber cache that will canonize cloud digests.
```

Comment:
No need for this note in the doc comment, can move it to the relevant spot in the impl if you want to keep it for now

That said, since this is in-progress, not a huge need to document the transient note